cypress/support/commands.js

```diff
@@ -120,31 +120,19 @@ function createEntity(registryName, apiKey, dataFile) {
 
 function deleteRegistry(registryName, apiKey){
 	cy.log('deleting registry...')
-	
+
 	cy.log('api-key = ' + apiKey)
-	let emptyUrl = '/registry/' + registryName + '/empty'
+
+	let url = '/registry/' + registryName
```

Comment:
Do we modify url?

Reply:
Changing to const, needs to be done across all tests